app/queries/admin/users_query.rb

```diff
@@ -10,7 +10,17 @@ class UsersQuery
     # @param search [String, nil]
     # @param roles [Array<String>, nil]
     # @param organizations [Array<String>, nil]
-    def self.call(relation: User.registered, role: nil, search: nil, roles: [], organizations: [])
+    # @param joining_start [String, nil]
+    # @param joining_end [String, nil]
+    # @param date_format [String]
+    def self.call(relation: User.registered,
+                  role: nil,
+                  search: nil,
+                  roles: [],
+                  organizations: [],
+                  joining_start: nil,
+                  joining_end: nil,
+                  date_format: "DD/MM/YYYY")
```

Comment:
Reading/sharing note:  In the controller, when we pass the parameter `date_format: nil` (e.g. `Admin::UsersQuery.call(date_format: params[:date_format])` where `params[:date_fromat] == nil`) we will not use the default "DD/MM/YYYY" as we explicitly passed a nil value.